docs/learning/guidelines/release-tags-guidelines.md

```diff
@@ -10,6 +10,8 @@ By default, any API is also usable by extensions, with the exception of APIs exp
 
 The `@deprecated` tag can be applied to any API to notify users that it may be removed or changed in a breaking way in the future. For `@public` APIs, deprecations must follow our [deprecation policy](../api-support-policies.md#api-deprecation-policy); marking `@beta`, `@alpha`, or `@internal` APIs as deprecated is optional, but recommended for giving users a heads-up.
 
+The `@preview` tag may be used alongside `@public` to indicate that the API is stable for the current major version, but may be removed or changed in a breaking way in the next major version. It provides a signal to users that the API is safe to use in production, but not yet committed for long-term stability beyond the current major version. The `@preview` tag is intended for features that are ready for broader adoption and feedback, but not yet finalized for long-term support.
```

Comment:
Why not change the definition of `@beta` instead of adding a new tag?

Reply:
afaik we havent downgraded any apis and their level of support from public to public preview
only new apis (primarily apis surrounding editing) have been promoted from beta to public preview
ignore the diff in this pr because the support for preview wasnt added before https://github.com/iTwin/itwinjs-core/pull/7903 merged

Reply:
well in that case you wouldn't have to downgrade anything from public to beta either, would you?

Reply:
alpha meant there was no documentation and provided even less stability than beta. instead of downgrading most of beta apis to alpha, we tok the approach of adding an additional tag in btw and promote to it without modifying any guarantees of stability users were already relying upon